fixtureeditor/editchannel.cpp

```diff
@@ -117,8 +116,7 @@ void EditChannel::init()
         m_presetCombo->setCurrentIndex(m_channel->preset());
     }
 
-    connect(m_presetCombo, SIGNAL(activated(int)),
-            this, SLOT(slotPresetActivated(int)));
+    connect(m_presetCombo, static_cast<void (QComboBox::*)(int)>(&QComboBox::activated), this, &EditChannel::slotPresetActivated);
```

Comment:
The cast is necessary here because there are two methods `QComboBox::activated`, and we have to let the compiler know which one we intend to target as signal.